performance-tests/bench_2/node_controller/main.cpp

```diff
@@ -442,8 +444,9 @@ int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
     std::cerr << "register_type failed for Report" << std::endl;
     return 1;
   }
+  type_name = report_ts->get_type_name();
   DDS::Topic_var report_topic = participant->create_topic(
-    report_topic_name, report_ts->get_type_name(), TOPIC_QOS_DEFAULT, nullptr,
+    report_topic_name,type_name, TOPIC_QOS_DEFAULT, nullptr,
```

Comment:
```suggestion
    report_topic_name, type_name, TOPIC_QOS_DEFAULT, nullptr,
```

Reply:
Why did you review yourself?

Reply:
It let me fix in with a few clicks in the GitHub web interface, instead of actually doing a commit in my repo.